src/main/java/com/ibm/kappnav/logging/Logger.java

```diff
@@ -66,33 +66,38 @@ public static void setLogLevel(String level) {
             level= level.toLowerCase(); 
             setLogTypes(false); 
             switch(level) { 
-               case "none": 
-                  break;               
-               case "error":
-                  enabled[LogType.ERROR.ordinal()]= true;  
-                  break;
-               case "warning": 
-                  enabled[LogType.ERROR.ordinal()]= true;  
-                  enabled[LogType.WARNING.ordinal()]= true;  
-                  break;
-               case "info": 
-                  enabled[LogType.ERROR.ordinal()]= true;  
-                  enabled[LogType.WARNING.ordinal()]= true;  
-                  enabled[LogType.INFO.ordinal()]= true;  
-                  break;
-               case "entry": 
-                  enabled[LogType.ERROR.ordinal()]= true;  
-                  enabled[LogType.WARNING.ordinal()]= true;  
-                  enabled[LogType.INFO.ordinal()]= true;  
-                  enabled[LogType.ENTRY.ordinal()]= true;  
-                  enabled[LogType.EXIT.ordinal()]= true;  
-                  break;
-               case "debug": 
-                  setLogTypes(true); 
-                  break;
-               case "all": 
-                  setLogTypes(true); 
-                  break;
+                case "none": 
+                    break;               
+                case "error":
+                    enabled[LogType.ERROR.ordinal()]= true;  
+                    break;
+                case "warning": 
+                    enabled[LogType.ERROR.ordinal()]= true;  
+                    enabled[LogType.WARNING.ordinal()]= true;  
+                    break;
+                case "info": 
+                    enabled[LogType.ERROR.ordinal()]= true;  
+                    enabled[LogType.WARNING.ordinal()]= true;  
+                    enabled[LogType.INFO.ordinal()]= true;  
+                    break;
+                case "entry": 
+                    enabled[LogType.ERROR.ordinal()]= true;  
+                    enabled[LogType.WARNING.ordinal()]= true;  
+                    enabled[LogType.INFO.ordinal()]= true;  
+                    enabled[LogType.ENTRY.ordinal()]= true;  
+                    enabled[LogType.EXIT.ordinal()]= true;  
+                    break;
+                case "debug": 
+                    setLogTypes(true); 
+                    break;
+                case "all": 
+                    setLogTypes(true); 
+                    break;
+                default : // same as info
```

Comment:
Could add a debug here noting the invalid level value requested.

Reply:
Adding debug msg.